python/MaterialXTest/main.py

```diff
@@ -481,6 +481,44 @@ def test_ReadXml(self):
         mx.readFromXmlFile(doc, filename, _searchPath)
         self.assertTrue(doc.validate()[0])
 
+    def test_WriteXml(self):
```

Comment:
Since we're already testing XML serialization in the `test_ReadXml` command above, and we already have access to the full set of data libraries in that context, perhaps it makes more sense to add a handful of new code blocks to the existing test, rather than creating a brand new one?

If it would add clarity, I think it would be fine to rename the existing command `test_XmlIo`, to reinforce the idea that it covers XML serialization as well.

Reply:
Should be simplified now to a single additional code block & did the test rename.